<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorDialog</class>
 <widget class="QDialog" name="MajorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="insert_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="major_tableWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Major</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Course</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="undo_pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redo_pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
